src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -596,6 +606,46 @@ const auto testParams2D_Brgemm_smoke = ::testing::Combine(fullyConnectedParams2D
 
 INSTANTIATE_TEST_SUITE_P(smoke_FC_2D_Brgemm, MatMulLayerCPUTest, testParams2D_Brgemm_smoke, MatMulLayerCPUTest::getTestCaseName);
 
+const std::vector<ShapeRelatedParams> IS2D_Brgconv1x1_smoke = {
```

Comment:
1. Why 3D case is not covered here?
2. Current impl support switches between IP and Conv1x1 on each inference based on actual shape. Is it covered here?
3. Conv1x1 might have different weights layout because of different batch size. Can we cover this case as well?

Reply:
Will add tests to cover all mentioned cases in a sperate PR.

Reply:
Testcase added in #14144 .